app/routes/contact/components/ContactForm.js

```diff
@@ -19,27 +21,62 @@ type Props = FieldProps & {
   addToast: ({ message: string }) => void,
   reset: (form: string) => void,
   change: (field: string, value: boolean) => void,
-  loggedIn: boolean
+  loggedIn: boolean,
+  groups: Array<Group>
 };
 
 const ContactForm = (props: Props) => {
-  const { invalid, pristine, submitting } = props;
+  const { invalid, pristine, submitting, groups } = props;
   const disabledButton = invalid || pristine || submitting;
 
   const submit = data => {
     return props
-      .sendContactMessage(data)
+      .sendContactMessage({
+        ...data,
```

Comment:
Does this make sense? Wrapping the spread version of the `data` object and setting the `recipient_group` as `data.recipient_group.value`.

Reply:
Not sure, but it works 😅 Switching their order might make more sense?

Reply:
I haven't looked at the `sendContactMessage` function, but it used to take an object, and now it takes the same object with one more value set to `data.recipient_group.value`. So in my mind, you could just send the data object and extract the `recipient_group.value` later, as it will be available. But if it works it's good :)